content/general/authentication.textile

```diff
@@ -7,68 +7,29 @@ languages:
 
-* the script, program or system holding the key is not exposed; for example, typically on one of your own servers. A key should not be embedded in a script in a public-facing web page
-* a secure, unmediated connection exists between the client and the Ably service. Keys should only really be sent over a TLS connection (that's either an HTTPS connection, or an encrypted realtime connection). A key should not be used over a proxied connection unless the proxy is trusted
-* access needs to be granted selectively to groups of clients to specific channels or channel namespaces, but only a small number of such access control groups need to be established
-* clients are trusted to assume any client ID in the operations they are permitted to perform
-* you don't wish to run your own server to control access
+Every application has one or more private API keys that you set up and "configure via the dashboard":https://support.ably.io/support/solutions/articles/3000030502-setting-up-and-managing-api-keys. These keys can be used to authenticate with Ably. It is recommended that private API keys are never shared with untrusted parties, and as such, should only be used by your trusted secure servers when authenticating with Ably.
 
-**Token authentication** is appropriate when:
-
-* there is a risk of exposure of the client's credentials, either directly or over an insecure, or insecurely proxied, connection
-* a client is only intended to have use of the service for a limited period of time
-* a client needs to have the ability to "identify itself":#identified-clients (authenticate its specific identity with a @clientId@), but cannot be trusted sufficiently not to masquerade as a different client identity
-* fine-grained access needs to be given on a per-client basis to specific channels and/or capabilities
-* there may be a requirement to be able to revoke the rights of a client individually
-
-Note that many applications will most naturally use a mixed strategy: one or more trusted application servers will use basic authentication to access the service and issue tokens over HTTPS, whereas remote browsers and devices will use individually issued tokens.
-
-Both types of authentication rely on a series of API keys to be set up for each application. Each key is "configured via the dashboard":https://support.ably.io/support/solutions/articles/3000030502-setting-up-and-managing-api-keys and exposed as a single string such as @{{API_KEY}}@ which contains information that identifies the key as well as containing the "secret" key value.
-
-h3(#basic-authentication). Basic Authentication explained
-
-Basic authentication is the simplest way to authenticate with Ably using an API key string when instancing the client library. It is as simple as:
+Basic authentication is the simplest way to authenticate with Ably. It is as simple as:
 
 bc[javascript](code-editor:authentication/basic-auth). var ably = new Ably.Realtime({ key: '{{API_KEY}}' });
 
```

Comment:
This section 'Process used' is not the same size as the subsections of Token Auth. Maybe make them more consistent? Is this intentional for some reason?